TypeCobol/Compiler/CodeElements/CodeElementEnd.cs

```diff
@@ -2,6 +2,6 @@
 
 	public abstract class CodeElementEnd: CodeElement {
 	    protected CodeElementEnd(CodeElementType type) : base(type) { }
-	}
+    }
```

Comment:
to revert

Reply:
@laabidiHend Can you check this comment.